src/statd/statd.c

```diff
@@ -221,56 +263,46 @@ static void sr_event_cb(struct ev_loop *, struct ev_io *w, int)
 	sr_subscription_process_events(sub->sr_sub, NULL, NULL);
 }
 
-static int sub_to_iface(struct statd *statd, const char *ifname)
+static int subscribe(struct statd *statd, char *model, char *xpath, const char *name,
+                     int (*cb)(sr_session_ctx_t *session, uint32_t, const char *,
+                               const char *, const char *, uint32_t,
```

Comment:
Indentation problem.
I believe the kernel way is
```
int long_function_nam(int foo,
<tab><tab><tab><space>int bar)
```
Where tabs are used first and if the second line doesn't line up with the bracket it's padded with spaces.

Reply:
Indeed. I've added `src/.clang-format` from the kernel, with some minor additions at the end.  It actually works.

Reply:
But it is a function pointer? Is it still valid to do so?

Reply:
Ah, of course, didn't see that at first glance. Shame on me.

Reply:
Right, I missed that too. Sorry :)